samples/assay_transform_function_example.py

```diff
@@ -0,0 +1,22 @@
+# Import the transform helper from the labkey python api library
+from labkey.utils import transform_helper
+
+# the run properties filepath must be defined in this way so that it can be fed into the transform helper
+filepath = '${runInfo}'
+
+# define the function that you are using to transform your results data array
+def transform(grid):
```

Comment:
I am trying to move us away from this samples folder, can you move this code example to a markdown file in the `/docs` folder?